areas.wotw

```diff
@@ -365,7 +376,19 @@ anchor MarshSpawn.PoolsBurrowsSignpost at -898, -4436:  # the waypoint between p
       Burrow, Sword
       Burrow, Hammer, Damage=10
   pickup MarshSpawn.LifepactShard:
-    moki: Regenerate, Damage=40, Combat=Hornbug+Bat+2xSandworm+2xLizard+3xSkeeto+SneezeSlug, Bash, DoubleJump OR Launch
+    moki, MarshSpawn.BurrowArena:
+      Combat=Bat+WeakSlug, DoubleJump OR Dash OR Launch
+      Bash, DoubleJump OR Dash OR Launch
+    gorlek, MarshSpawn.BurrowArena:
+      Damage=10, Sword  # Use sword to combat slug/bat
+      Damage=10, Glide, Combat=Bat+WeakSlug OR Bash
+    kii, MarshSpawn.BurrowArena:
+      Sword OR Hammer  # Use weapon to combat slug/bat
```

Comment:
Killing the bat damageless with just a melee weapon seems not kii either, kii could run past and take damage though